src/dbnode/storage/namespace.go

```diff
@@ -1045,6 +1045,9 @@ func (n *dbNamespace) Bootstrap(
 		}
 		if !bootstrapped {
 			// NB(r): Not bootstrapped in this bootstrap run.
+			n.log.Warn("skipping shard",
```

Comment:
Nit - consider adding some kind of reason to the log message for clarity/peace of mind, such as:
```suggestion
			n.log.Warn("skipping not bootstrapped shard",
```